cpl/module_inline.F90

```diff
@@ -0,0 +1,260 @@
+module module_inline
+!
+!*** this module contains the subroutines for cdeps inline capability 
+!
+! revision history
+!  22 Aug 2023: U. Turuncoglu  Initial development
+!
+  use ESMF
+  use dshr_mod        , only: dshr_pio_init
+  use dshr_strdata_mod, only: shr_strdata_type
+  use dshr_strdata_mod, only: shr_strdata_init_from_inline
+  use dshr_strdata_mod, only: shr_strdata_advance
+  use dshr_stream_mod , only: shr_stream_init_from_esmfconfig
+
+  use GFS_typedefs    , only: kp => kind_phys
+  use CCPP_data       , only: GFS_control
+  use atmos_model_mod , only: setup_inlinedata
+
+  implicit none
+
+  private
+  public stream_init
+  public stream_run
+
+  type(ESMF_Grid) :: grid
+  type(ESMF_Mesh) :: mesh
+  type(ESMF_Field) :: fgrid
```

Comment:
To me it's a little confusing to name variable of type Field, fgrid. Could this just be named `field`, like mesh is `mesh`, and grid is `grid`

Reply:
Ok, I now see that `fgrid` here (probably) means field on a grid, since I see local variable `fmesh`, which is probably field on a mesh.

What's the reason for even providing a field on a mesh only to redistributing in to field on a grid? Is there a reason cdeps provides (reads) field on a mesh instead of a field on a grid?